src/services/services.ts

```diff
@@ -4473,9 +4514,12 @@ module ts {
                             if ((findInStrings && isInString(position)) ||
                                 (findInComments && isInComment(position))) {
                                 result.push({
-                                    fileName: sourceFile.fileName,
-                                    textSpan: createTextSpan(position, searchText.length),
-                                    isWriteAccess: false
+                                    definition: undefined,

```

Comment:
Explain that the only caller who cares is rename. It doesn't use definitions, and it doesn't mind if multiple definitions are undefined.